jekyll/_cci2/server/Installation/2-install-core.adoc

```diff
@@ -23,215 +23,300 @@ NOTE: In the following sections replace any items or credentials displayed betwe
 
-=== GitHub
+. Do nothing.  https://letsencrypt.org/[Let's Encrypt] will automatically request and manage certificates for you.  This is a good option for trials, it is not recommended for production use.
 
-You created your Github OAuth application in the prerequisite phase. Use the data to complete the following settings:
+. You can supply a private key and certificate
 
-.GitHub Settings
-image::server-3-github-settings.png[Screenshot showing GitHub settings]
+You may have created this during the prerequisite steps. You can retrieve the values with the following commands:
+```bash
+cat /etc/letsencrypt/live/<CIRCLECI_SERVER_DOMAIN>/privkey.pem
+cat /etc/letsencrypt/live/<CIRCLECI_SERVER_DOMAIN>/fullchain.pem
+```
 
-* *GitHub Type (required)* -
-Select Cloud or Enterprise (on premises).
+Add them to values.yaml:
+```
+tls:
+  certificate: "<full-chain>"
+  privateKey: "<private-key>"
+```
+
+[start=3]
+. Have https://docs.aws.amazon.com/acm/latest/userguide/acm-overview.html[AWS Certificate Manager (ACM)] automatically request and manage certificates for you.  Follow the https://docs.aws.amazon.com/acm/latest/userguide/gs-acm-request-public.html[ACM documentation] for instructions on how to generate ACM certificates.
 
-* *OAuth Client ID (required)* -
-The OAuth Client ID provided by GitHub.
+Enable `aws_acm` and add the `service.beta.kubernetes.io/aws-load-balancer-ssl-cert` annotation to point at the ACM ARN
+```
+nginx:
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-ssl-cert: <acm-arn>
+  aws_acm:
+    enabled: false
+```
 
-* *OAuth Client Secret (required)* -
-The OAuth Client Secret provided by GitHub.
+[WARNING]
+==== 
+If you have already deployed CircleCI server, enabling ACM is a destructive change to the loadbalancer. The service will have to be regenerated to allow the use of your ACM certificates and so the associated loadbalancer will also be regenerated. 
+You will need to update your DNS records to the new loadbalancer once you have redeployed CircleCI server.
+====
 
-* *Github Enterprise Fingerprint* -
-Required when using a proxy. Include the output of `ssh-keyscan github.example.com` in the text field.
+[start=4]
+. Disable TLS termination within CircleCI.  The system will still need to be accessed over HTTPS, so TLS termination will be required somewhere upstream of CircleCI.  Implement this by following step 1 (do nothing) and forward to CircleCI on port 80 after terminating TLS.
 
-=== Object storage
 
-You created your Object Storage Bucket and Keys in the prerequisite steps. Use the data to complete the following settings depending on your platform.
+=== GitHub
 
-.Object Storage Settings
-image::server-3-object-storage.png[Screenshot showing object storage settings]
+You created your Github OAuth application in the prerequisite phase. Add the client ID and secret.
 
-// Don't include this section in the GCP PDF.
+```
+github:
+  clientId: <client-id>
+  clientSecret: <client-secret>
```

Comment:
these will also go in secret with [SERVER-1833](https://circleci.atlassian.net/browse/SERVER-1833)